tests/Integration/Behaviour/Features/Context/Domain/OrderCartFeatureContext.php

```diff
@@ -85,9 +85,9 @@ public function thereIsDuplicatedCartForCart(string $duplicatedCartReference, st
         $cartId = SharedStorage::getStorage()->get($cartReference);
 
         /** @var CartInformation $cartInformation */
-        $cartInformation = $this->getQueryBus()->handle(new GetCartInformation($cartId));
+        $cartInformation = $this->getQueryBus()->handle(new GetCartForOrderCreation($cartId));
```

Comment:
Maybe it's overkill but I guess we should test the result in both mode (with split gifts, and without split gifts)

Reply:
Reminder perform two checks here:
- one with `GetCartForOrderCreation::hideDiscounts = false`
- one with `GetCartForOrderCreation::hideDiscounts = true`